### PR TITLE
Phpsu 2247 upgrade dependencies

### PR DESCRIPTION
**Pull request by tiotDak**

### CODE REVIEW

File: app/Services/UserService.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+class UserService
+{
+    /**
+     * @return bool|array
+     */
+    public function checkAccess()
+    {
+        if (Auth::guard('api')->user()->id != request()->route('record')->getAttributes()['id']) {
+            $err['forbidden'] =  'You don\'t have to access this resource.';
```
Comment (tatyana-ak) [inline app/Services/UserService.php]: Maybe something like that: "You don't have permission to access this resource" ?

File: stubs/JsonApi/Users/Validators.php [hunk 2]
```diff
@@ -56,7 +56,7 @@ class Validators extends AbstractValidatorProvider
     protected function attributeRules($record = null)
     {
         $email = 'required|email|max:255|unique:users';
-        $password = 'required|min:8|max:16|complexity_password';
+        $password = 'required|min:8|max:16';
```
Comment (tatyana-ak) [inline stubs/JsonApi/Users/Validators.php]: Why do you remove this check?

File: .env.example [hunk 3]
```diff
@@ -11,6 +11,12 @@ DB_DATABASE=homestead
 DB_USERNAME=homestead
 DB_PASSWORD=secret
 
+TESTING_DB_HOST=127.0.0.1
+TESTING_DB_PORT=3306
+TESTING_DB_DATABASE=homestead_testing
+TESTING_DB_USERNAME=homestead
+TESTING_DB_PASSWORD=secret
```
Comment (tatyana-ak) [inline .env.example]: Probably it's will be better to move this setting to .env.testing without TESTING_ prefix

File: stubs/Controllers/UsersController.stub [hunk 4]
```diff
@@ -2,71 +2,168 @@
...
-     *   tags={"User actions"},
-     *   summary="Filter users with limited per page quantity.",
-     *   description="Filter users",
+     * Display a listing of the resource.
+     *
+     * @SWG\Get(
+     *   path="/users",
+     *   tags={"Users"},
+     *   summary="get me",
+     *   description="get me",
```
Comment (tatyana-ak) [inline stubs/Controllers/UsersController.stub:49]: Does the method return only one logged user?

File: app/Http/Controllers/Api/v1/UsersController.php [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+     *         property="data",
+     *         type="object",
+     *         @SWG\Property(property="type", type="string", default="users", example="users"),
+     *         @SWG\Property(
+     *           property="attributes",
+     *           type="object",
+     *           @SWG\Property(property="email", type="string", example="user@mail.com", description="required"),
+     *           @SWG\Property(property="password", type="string", example="password", description="required"),
+     *           @SWG\Property(property="name", type="string", example="Steven", description="required"),
+     *         )
```
Comment (tatyana-ak) [inline app/Http/Controllers/Api/v1/UsersController.php]: Is it possible to set "activated" property by POST request?

File: stubs/Controllers/UsersController.stub (hunk 4 above)
Comment (tiotDak) [reply stubs/Controllers/UsersController.stub:49]: yes

File: app/Http/Controllers/Api/v1/UsersController.php (hunk 5 above)
Comment (tiotDak) [reply app/Http/Controllers/Api/v1/UsersController.php]: yes

File: app/Services/UserService.php (hunk 1 above)
Comment (tiotDak) [reply app/Services/UserService.php]: ok

File: .env.example (hunk 3 above)
Comment (tiotDak) [reply .env.example]: ok

**Pull request merged**